### DISCUSSION TITLE
Rate To High

### DISCUSSION THREAD

**Discussion by ma-gu-16** [jomjol/AI-on-the-edge-device#2574] [Q&A]
Mal sind die Rate zu High und dann wieder zu Low mit einem N im Wert ? Hat jemand eine Ahnung was man besser machen könnte ? 

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/22169074/26ef49e3-4309-40f3-8668-aff79f117280)

`

- [2d12h06m10s] 2023-08-14T03:10:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d12h11m10s] 2023-08-14T03:15:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d12h16m10s] 2023-08-14T03:20:17 <ERR> [POSTPROC] main: Raw: 183.5724, Value: , Status: Rate too high - Read: 183.5724 - Pre: 180.8259 - Rate: 2.7465
- [2d12h21m10s] 2023-08-14T03:25:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d12h26m10s] 2023-08-14T03:30:17 <ERR> [POSTPROC] main: Raw: 17N.5724, Value: , Status: Neg. Rate - Read: - Raw: 17N.5724 - Pre: 180.8259
- [2d12h31m10s] 2023-08-14T03:35:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d12h36m10s] 2023-08-14T03:40:17 <ERR> [POSTPROC] main: Raw: 17N.5724, Value: , Status: Neg. Rate - Read: - Raw: 17N.5724 - Pre: 180.8259
- [2d12h41m10s] 2023-08-14T03:45:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d12h46m10s] 2023-08-14T03:50:17 <ERR> [POSTPROC] main: Raw: 183.5724, Value: , Status: Rate too high - Read: 183.5724 - Pre: 180.8259 - Rate: 2.7465
- [2d12h51m10s] 2023-08-14T03:55:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d12h56m10s] 2023-08-14T04:00:17 <ERR> [POSTPROC] main: Raw: 17N.5724, Value: , Status: Neg. Rate - Read: - Raw: 17N.5724 - Pre: 180.8259
- [2d13h01m10s] 2023-08-14T04:05:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d13h06m10s] 2023-08-14T04:10:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d13h11m10s] 2023-08-14T04:15:17 <ERR> [POSTPROC] main: Raw: 183.5724, Value: , Status: Rate too high - Read: 183.5724 - Pre: 180.8259 - Rate: 2.7465
- [2d13h16m10s] 2023-08-14T04:20:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d13h21m10s] 2023-08-14T04:25:17 <ERR> [POSTPROC] main: Raw: 17N.5724, Value: , Status: Neg. Rate - Read: - Raw: 17N.5724 - Pre: 180.8259
- [2d13h26m10s] 2023-08-14T04:30:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d13h31m10s] 2023-08-14T04:35:17 <ERR> [POSTPROC] main: Raw: 183.5724, Value: , Status: Rate too high - Read: 183.5724 - Pre: 180.8259 - Rate: 2.7465
- [2d13h36m10s] 2023-08-14T04:40:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d13h41m10s] 2023-08-14T04:45:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d13h46m10s] 2023-08-14T04:50:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d13h51m09s] 2023-08-14T04:55:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d13h56m10s] 2023-08-14T05:00:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259
- [2d14h01m09s] 2023-08-14T05:05:17 <ERR> [POSTPROC] main: Raw: 180.5724, Value: , Status: Neg. Rate - Read: - Raw: 180.5724 - Pre: 180.8259

`

**Comment by friedpa** [comment]
Allgemeines zum Bild: zuviele Reflexionen, der Marker rechts dürfte auch nicht passen. Die Frames um die Ziffern entsprechen nicht den Vorgaben, siehe WIKI (versuche die Optik näher an den Zähler zu bringen). Nach einer Optimierung sollte das so aussehen, dann hat man fast kaum noch Fehlmessungen:

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/42890695/0a1c4ff9-5aae-4319-a2b2-a7d354c11c3b)

**Reply by ma-gu-16** [reply]
Ok das sieht komplett anderst aus,  ich versuche mal zuerst zu optimieren. Danke dir.

**Comment by friedpa** [comment]
Nur noch eine Frage: hat Dein Wasserzähler wirklich eine IEC M-Bus Schnittstelle, wie auf dem Wasserzähler beschriftet? Wann ja, wäre es ja viel besser (und stabiler) den M-Bus auszulesen..... z.B. mit dem IOBroker oder Homeassistant

**Reply by ma-gu-16** [reply]
Keine Ahnung :) ob und wie. Ich sehe zumindest kein Stecker o.ä.  Ich informiere mich mal, danke für den Tipp

**Reply by ma-gu-16** [reply]
Ok laut Datenblatt hat der wohl so etwas :) Dann werde ich diesen weg gehen. Herzlichen Dank für den Hinweis. 

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/22169074/2806722d-f795-4df9-b6ec-ddfe10cea981)

**Reply by ma-gu-16** [reply]
Es wäre tatsächlich ein Mbus vorhanden, jedoch verblombt. nach Absprache mit den Wasserwerken wollen diese den MBUS in naher Zukunft selbst nutzen für die Fernablesung. Somit darf ich diesen nicht verwenden :( Also wieder zurück zur AI Cam.

**Comment by ma-gu-16** [comment]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/22169074/4d50b3d8-017b-400b-813e-c3f1c33196fa)

Nun erkennt er wohl die 1 als eine 7 , kann ich irgendwie manuell einfluss darauf nehmen ?
